internal/pkg/values/scope.go

```diff
@@ -225,7 +224,7 @@ func (s Scopes) GetDownscaleReplicas() (int32, error) {
 // GetScaleChildren gets the scale children of the first scope that implements scale children.
 func (s Scopes) GetScaleChildren() bool {
 	for _, scope := range s {
-		if scope.ScaleChildren {
+		if scope.ScaleChildren.isSet && scope.ScaleChildren.value {
 			return true
 		}
```

Comment:
It would actually be this, otherwise the tristatebool wouldn't make a difference here

```suggestion
		if scope.ScaleChildren.isSet {
			return scope.ScaleChildren.value
		}
```